bigquery/src/test/java/org/apache/zeppelin/bigquery/BigQueryInterpreterTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
I think this license header is still different from a [ASF standard one](http://www.apache.org/legal/src-headers.html#headers) (does not mention NOTICE, have a bit different wording)